src/proxy/momento/src/admin.rs

```diff
@@ -16,7 +16,6 @@ pub static ADMIN_CONN_CLOSE: Counter = Counter::new();
 
 pub(crate) async fn admin(mut log_drain: Box<dyn logger::Drain>, admin_listener: TcpListener) {
     loop {
-        clocksource::refresh_clock();
```

Comment:
Just to be sure: is this no longer necessary with the new version of clocksource?

Reply:
Correct. We no longer have any API that returns a "recent" (cached) time.